source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,128 @@
 Resolver for Home Networks
+==========================
+
```

Comment:
General note: I would prefer anything that resides on the system (e.g., files, commands) and is used inline in text to be formatted as inline literal; that is double back ticks.